test/CMakeLists.txt

```diff
@@ -2220,32 +2216,6 @@ endif()
 # Land product converters
 #==============================================================================
 
-if( iodaconv_pygrib_ENABLED )
-  ecbuild_add_test( TARGET  test_${PROJECT_NAME}_ims_scf
-                    TYPE    SCRIPT
-                    ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
-                    COMMAND bash
-                    ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
-                            netcdf
-                            "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/ims_scf2ioda.py
-                            -i testinput/imssnow_24km.grib2
```

Comment:
@srherbener this input file wasn't in the `testinput` directory does it only get loaded if the `USE_PYGRIB` is set?  does it have to be removed from somewhere

basically all the `testinput` and `testoutput` files are not in the repository so as you can see they are not removed in this PR

Reply:
That seems another indication that these converters are not being used.